doc/manual/src/installation/index.md

```diff
@@ -14,6 +14,10 @@ This option requires either:
 * Linux running systemd, with SELinux disabled
 * MacOS
 
+> **Caution**
+>
+> macOS users: If you recently updated to macOS 15 Sequoia and are seeing `error: the user '_nixbld1' in the group 'nixbld' does not exist` errors when you invoke Nix commands, see https://github.com/NixOS/nix/issues/10892 for instructions about how to fix your install without reinstalling.
```

Comment:
```suggestion
> **Updating to macOS 15 Sequoia**
>
> If you recently updated to macOS 15 Sequoia and are getting
> ```console
> error: the user '_nixbld1' in the group 'nixbld' does not exist
> ```
> when running Nix commands, refer to GitHub issue [NixOS/nix#10892](https://github.com/NixOS/nix/issues/10892) for instructions to fix your installation without reinstalling.
```

Reply:
Not certain if it's what you intended, but latest force-push updates all 3 sites to use this suggestion.